### PR TITLE
Fixes broken regression suite specs

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

HOTT-<TODO>

### What?

I have added/removed/altered:

- [x] Fix change in quotas for Morocco and use Israel instead
- [x] Fix change in measure sid for XI MFN duty

### Why?

I am doing this because:

- These reflect changes in the underlying data that's returned from both apis

### CODE REVIEW

File: cypress/integration/UK/FE/e2eSpecialComms-UK.spec.js [hunk 1]
```diff
@@ -56,18 +56,17 @@ describe('🇬🇧 💡 | e2eSpecialComms-UK | Select Commodities and measure de
     cy.contains('8.00 %');
   });
 
-  it(' 🍅 Cherry Tomatoes 🇲🇦 \n- Check that the duty is a standard ad valorem (8.00%) ,Check that there is no condition against the ad valorem third country duty,Check that there is a single quota for Morocco', function() {
-    cy.visit('/commodities/0702000007?country=MA#import');
+  it.only(' 🍅 Cherry Tomatoes 🇲🇦 \n- Check that Israel has a preferential tariff quota', function() {
```
Comment (jebw) [inline cypress/integration/UK/FE/e2eSpecialComms-UK.spec.js]: This will exclude other specs in this file?

Comment (willfish) [inline cypress/integration/UK/FE/e2eSpecialComms-UK.spec.js:59]:
```suggestion
  it(' 🍅 Cherry Tomatoes 🇲🇦 \n- Check that Israel has a preferential tariff quota', function() {
```

**Review by jebw — COMMENTED** [review]
Looks good but needs the `it.only` dropping

**Review by jebw — APPROVED** [review]
LGTM

**Pull request merged**